connectors/sources/jira.py

```diff
@@ -320,8 +371,310 @@ def get_default_configuration(cls):
+                    access_control.add(
+                        _prefix_account_id(
+                            user_id=actor.get("actorUser").get("accountId")
+                        )
+                    )
+                    access_control.add(_prefix_username(user=actor.get("displayName")))
+                elif actor.get("actorGroup"):
+                    access_control.add(
+                        _prefix_group_id(
+                            group_id=actor.get("actorGroup").get("groupId")
+                        )
+                    )
+            yield access_control
+
+    async def _issue_access_control(self, issue_key, project):
+        if not self._dls_enabled():
+            return []
+
+        access_control = set()
+        async for response in self._issue_security_level(issue_key=issue_key):
+            if security := response.get("fields", {}).get("security"):
+                level_id = security.get("id")
+                async for members in self._issue_security_level_members(
+                    level_id=level_id
+                ):
+                    for actor in members["values"]:
+                        if actor.get("holder", {}).get("type") == "user":
+                            user = actor.get("holder", {}).get("user", {})
+                            if self._is_active_atlassian_user(user_info=user):
+                                access_control.add(
+                                    _prefix_account_id(user_id=user.get("accountId"))
+                                )
+                                access_control.add(
+                                    _prefix_username(user=user.get("displayName"))
+                                )
+                        elif actor.get("holder", {}).get("type") == "group":
+                            group_id = (
+                                actor.get("holder", {}).get("group", {}).get("groupId")
+                            )
+                            access_control.add(_prefix_group_id(group_id=group_id))
+                        elif actor.get("holder", {}).get("type") == "projectRole":
+                            if (
+                                role_id := actor.get("holder", {})
+                                .get("projectRole", {})
+                                .get("id")
+                            ):
+                                # Project Role - `atlassian-addons-project-access` with id 10003 is not needed for DLS
+                                if role_id == 10003:
+                                    continue
+                                access_control = await anext(
+                                    self._project_role_members(
+                                        project=project,
+                                        role_id=role_id,
+                                        access_control=access_control,
+                                    )
+                                )
```

Comment:
Small readability nit:

```suggestion
                                is_addons_projects_access = role_id == 10003
                                if not is_addons_projects_access:
                                  access_control = await anext(
                                      self._project_role_members(
                                          project=project,
                                          role_id=role_id,
                                          access_control=access_control,
                                      )
                                  )
```